google-http-client/src/main/java/com/google/api/client/http/HttpTransport.java

```diff
@@ -158,4 +158,13 @@ public boolean isMtls() {
    * @since 1.4
    */
   public void shutdown() throws IOException {}
+
+  /**
+   * Default implementation of whether the http transport is shutdown or not.
```

Comment:
The Javadoc for methods in abstract classes should focus on the expected behavior, as the subclasses will be referring to it if an overriding is needed, e.g. [ExecutorService.isShutdown()](https://docs.oracle.com/javase%2F7%2Fdocs%2Fapi%2F%2F/java/util/concurrent/ExecutorService.html#isShutdown()). We can mention the default implementation but should not be the focus.

Reply:
Done, thanks for the suggestion.